pkg/util/deployment/deployment.go

```diff
@@ -28,8 +28,13 @@ import (
 	podutil "k8s.io/kubernetes/pkg/util/pod"
 )
 
+const (
+	// The revision annotation of a deployment's replication controllers which records its rollout sequence
+	RevisionAnnotation = "deployment.kubernetes.io/revision"
```

Comment:
> You mean is not the correct place?

Sorry for being ambiguous. I meant that this is the correct place to ask because this is the original PR making the change.